Doc/library/subprocess.rst

```diff
@@ -512,6 +512,27 @@ functions.
    .. versionchanged:: 3.2
       *start_new_session* was added.
 
+   If *groups* not null, the setregid() system call will be made in the child
+   process prior to the execution of the subprocess. If the provided value is
+   a string, the provided group name will be looked up via
+   :func:`grp.getgrgid()` and the value in `gr_gid` will be used. If *groups*
+   is a list, strings will be looked up via :func:`grp.getgrgid()` and the
+   values in `gr_gid` will be used. Integer values will be passed verbatim.
+   The first item in the list will be passed go setregid(), and the following
+   items will be passed to the setgroups() system call. On sytems without
+   setgroups(), the remaining items will be discarded.  (POSIX only)
```

Comment:
which part is POSIX only?  if the entire feature is posix only, use the `.. availability:: POSIX` marker as seen elsewhere in docs before the versionadded marker.

same comment below.

Reply:
All of it is POSIX only, since `setgroups()` only exists on POSIX. Will add the `.. availability:: POSIX` line

Reply:
To be more clear, `setgroups()` and `setregid()` are only available on POSIX, so the feature is all POSIX only. Some POSIX systems don't have `setgroups()`, which is the reason for the fallback.